pkg/services/ngalert/api/api_prometheus.go

```diff
@@ -499,14 +501,7 @@ func toRuleGroup(log log.Logger, manager state.AlertInstanceManager, groupKey ng
 				}
 				alertingRule.State = "firing"
```

Comment:
We can do the same for this logic too, by just adding more fields to the status struct. If we do this enough, the API layer could become very thin.

This PR only handles the type-agnostic fields for now - basically only enough to build `apimodels.Rule` without consulting states directly. More can be considered in the future.